release_notes/Service_Fabric_ReleaseNotes_81CU2.md

```diff
@@ -0,0 +1,83 @@
+    * The simplest mitigation, when possible, is to delete and recreate the application in SF 7.0.<br/>
+    * The other option is to upgrade the application in SF 7.0 (for example, with a version only change).<br/>
+
+    If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+* .NET Core runtime LTS 2.1 runtime is now out of support as of Aug 21, 2021. Service Fabric releases after that date will drop support for Service Fabric apps running with .NET Core 2.1 runtime. This includes the following release versions and beyond - 8.0 CU4 and above, 8.1 CU2 and above and 8.2 RTO release and above. Current apps running on .NET Core 2.1 runtime will continue to work, but requests for investigations or request for changes will no longer be entertained. Service Fabric .NET SDK will take a dependency on .NET runtime 3.x features to support Service Fabric .NET Core apps. This has no impact on Service Fabric .NET Framework SDK.
+
+
+## Upcoming Breaking Changes
+* Ubuntu 16.04 LTS reached it's 5-year end-of-life window on April 30, 2021 and is no longer supported by it's vendors. Service Fabric runtime will drop support for Ubuntu 16.04 after that date as well. Current applications running on it will continue to work, but requests for investigations or requests for change will no longer be entertained. We recommend moving your applications to Ubuntu 18.04.
+
+
+## Service Fabric Common Bug Fixes
+
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 8.1.329.9590<br>Ubuntu 16 - 8.1.328.1<br>Ubuntu 18 - 8.1.328.1** | **Bug** | UpdateConfiguration API only sends Secondary replica role | **Brief desc**: As part of the UpdateConfiguration API call, Service Fabric runtime, was only sending Secondary replica role. <br> **Fix**: As part of the UpdateConfiguration API call, Service Fabric runtime will now send Auxiliary replica role to the replicator as well. |
+| **Windows - 8.1.321.9590<br>Ubuntu 16 - 8.1.323.1<br>Ubuntu 18 - 8.1.323.1** | **Bug** | RecoverPrimaryAsyncOperation causes endless rerun of UpdateServiceRequest with unchanged DNS name | **Brief desc**: NamingService would endlessly retry operations where DnsName is unchanged because the transaction would attempt to update and delete the key in the same transaction. <br> **Impact**: Naming StoreService may perpetually hold on to rejected operations and requeue them to the job queue. <br> **Fix**: Filter out updating DNS name when there is no change to shortcut the commit and avoid the code path that triggers the transaction conflict. <br> **Workaround**: Not applicable for Azure clusters. For others, do not supply the DnsName as part of Service Update request. To break the retry loop, change the DNS name and allow the repeating operation to reset this. If it does not, change the DNS name manually.  |
```

Comment:
Do we have any TSG or documentation on how to change the DNS name manually?

Reply:
No, not sure. I will have to check.